ads/google/imaVideo.js

```diff
@@ -1355,33 +1233,18 @@ function onFullscreenChange(global) {
 
 /**
  * Show a subset of controls when ads are playing.
- * Visible controls are countdownDiv, playPauseDiv, muteUnmuteDiv, and fullscreenDiv
+ * See CSS for selectors affected by -controls-ads and -controls-ads-mini
  *
  * @visibleForTesting
  */
 export function showAdControls() {
-  const hasMobileStyles = videoWidth <= 400;
-  const isSkippable = currentAd ? currentAd.getSkipTimeOffset() !== -1 : false;
-  const miniControls = hasMobileStyles && isSkippable;
-  // hide non-ad controls
-  [timeDiv, progressBarWrapperDiv].forEach((button) => {
-    setStyle(button, 'display', 'none');
-  });
-  // set ad control styles
-  setStyles(controlsDiv, {
-    'height': miniControls ? '20px' : '30px',
-    'justify-content': 'flex-end',
-    'padding': '10px',
-  });
-  [fullscreenDiv, playPauseDiv, muteUnmuteDiv].forEach((button) => {
-    setStyles(button, {'height': miniControls ? '18px' : '22px'});
-  });
-  setStyles(muteUnmuteDiv, {'margin-right': '10px'});
-  // show pause button while ad begins playing
-  changeIcon(playPauseDiv, 'pause');
-  // show ad controls
-  setStyle(countdownWrapperDiv, 'display', 'flex');
   showControls(true);
+  toggleRootDataAttribute('playing', true);
```

Comment:
Setting style here was because we would duplicate properties across state handlers for reversal. Instead, we set state: an `ad` is `playing` and it might be `skippable`.